airbyte-cdk/python/unit_tests/sources/file_based/file_types/test_avro_parser.py

```diff
@@ -142,17 +142,17 @@
                      id="test_decimal_missing_precision"),
         pytest.param(_default_avro_format, {"type": "bytes", "logicalType": "decimal", "precision": 9}, None, ValueError,
                      id="test_decimal_missing_scale"),
-        pytest.param(_default_avro_format, {"type": "bytes", "logicalType": "uuid"}, {"type": ["null", "string"]}, None, id="test_uuid"),
-        pytest.param(_default_avro_format, {"type": "int", "logicalType": "date"}, {"type": ["null", "string"], "format": "date"}, None,
+        pytest.param(_default_avro_format, {"type": "bytes", "logicalType": "uuid"}, {"type": "string"}, None, id="test_uuid"),
```

Comment:
This test was passing only if `avro_all_types_scenario` would be executed before as it would modify the values in memory. To fix this, I had to:
* Modify the test expected results
* Create a deepcopy of the dict that is returned [here](https://github.com/airbytehq/airbyte/pull/30453/files#diff-e07a7a23c8203f8a61e7c9d31cb495d6d9f8c7e1356ddff25630e704431d404cR214-R215)